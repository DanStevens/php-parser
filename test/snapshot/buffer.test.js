const parser = require("../../src/index");

describe("Test buffers", function() {
  it("handles buffers as input", function() {
    var buffer = new Buffer('echo "test"');
    expect(parser.parseEval(buffer)).toMatchSnapshot();
  });
});
