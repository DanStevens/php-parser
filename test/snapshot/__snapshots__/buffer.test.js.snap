// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test buffers handles buffers as input 1`] = `
Program {
  "children": Array [
    Echo {
      "arguments": Array [
        String {
          "isDoubleQuote": true,
          "kind": "string",
          "raw": "\\"test\\"",
          "value": "test",
        },
      ],
      "kind": "echo",
      "shortForm": false,
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;
